package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "eslint": "eslint --ignore-path .gitignore .",
     "eslint-fix": "eslint --fix --ignore-path .gitignore .",
-    "stylelint": "stylelint --ignore-path .gitignore .",
+    "stylelint": "stylelint src/style/*.css",
```

Comment:
Can you also remove all the `ignore` portion of the stylelintrc file and still have everything work?  I suspect so, but please try and update this if so.

Reply:
I will remove the ignore portion in stylelintrc file.